recipes/iiboost/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "iiboost" %}
+{% set version = "0.2" %}
+{% set sha256 = "0ecd906b453a6a278651e324352bd6f820b70aa4ca15fa42843bdb5c01ee7079" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/cbecker/iiboost/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+  patches:
+    - cmath_fixes.diff
+    - python3.diff
+    - msvc_fixes.diff
+
+build:
+  number: 0
+  skip: True        # [np!=112 or (win and py27)]
+  features:
+    - vc14          # [win and py>=35]
+
+requirements:
+  build:
+    - python
+    - toolchain
+    - gcc                   # [linux]
+    - cmake
+    - eigen
+    - numpy x.x
+    - itk       4.11.*
+    - boost     1.63.*
```

Comment:
There seem to be a few other build and run requirements not included in this list. Looks like they may be transient requirements. For example, I'm seeing `zlib`, `expat`, `hdf5`, `jpeg`, `libpng`, `gdcm` just by looking at the linkages fixed on the Travis CI build. Would recommend adding these and [pinning]( https://github.com/conda-forge/conda-forge.github.io/blob/808c6f610acfe910fe4a50ce4ddd58cc8a202f4c/scripts/pin_the_slow_way.py#L29-L70 ) them. Also for any that don't have a pinning, it would be good to look into a reasonable pinning. We often turn to [ABI Laboratory]( https://abi-laboratory.pro/ ) for this sort of thing. Here is the history on [GDCM]( https://abi-laboratory.pro/tracker/timeline/gdcm/ ) for example.

Note: This appears to be [linking]( https://travis-ci.org/conda-forge/staged-recipes/builds/231676573#L2334 ) to the C++ interface of HDF5. ( cc @stuarteberg )